.github/workflows/build.yml

```diff
@@ -10,10 +10,14 @@ on:
     inputs:
       keep_workdir:
         required: false
-      base-distro:
-        description: Use one of the entries in distro-defs.yml, currently centos-stream-8-copr centos-stream-9-copr centos-stream-8-cbs centos-stream-9-cbs
+      distro-os:
```

Comment:
Perhaps 'excluded-distro-os'? It serves a different function from distro-os in the matrix (and in the Makefile).